python/lsst/ctrl/mpexec/cli/opt/options.py

```diff
@@ -446,3 +446,9 @@ def _to_int(value):
     ),
     default="all",
 )
+
+job_report_option = MWOptionDecorator(
+    "--job-report",
+    help="Location for storing job report (JSON file).",
```

Comment:
Even though this was requested for running PanDA jobs, I think there will be folks who start looking at these reports as soon as they discover the command line option.  Is there a standard way to warn that the contents in the file are still in flux?  (And if we're going to change the "job" to something else it would be nice to do it before these folks find this option.)

Reply:
I can add a warning to the help string, I don't want to issue an actual WARNING for it. Let's try to decide on a name during the meeting.

Reply:
Command line option was renamed to `--summary`.